target/linux/ramips/image/mt7688.mk

```diff
@@ -34,3 +34,21 @@ define Device/widora-neo
   DEVICE_PACKAGES := kmod-usb2 kmod-usb-ohci
 endef
 TARGET_DEVICES += widora-neo
+
+define Device/Omega2
```

Comment:
Lowercase omega2 here and in TARGET_DEVICES please. you can drop the SUPPORTED_DEVICES afterwards. Same applies to omega2p.